src/main/cms/cms_menu_imu.c

```diff
@@ -190,7 +205,7 @@ static const void *cmsx_PidWriteback(displayPort_t *pDisp, const OSD_Entry *self
         pidProfile->pid[i].O = tempPidO[i];
         pidProfile->pid[i].B = tempPidB[i];
     }
-    pidInitProfile(currentPidProfile);
+    changePidProfile(pidProfileIndex);
```

Comment:
You MUST NOT try to change the PID profile from CMS.
It will go out-of-sync with the Tx control.